projects/packages/jetpack-mu-wpcom/src/features/wpcom-site-menu/wpcom-site-menu.php

```diff
@@ -438,6 +438,10 @@ function wpcom_add_hosting_menu_intro_notice() {
 			height: 16px;
 			margin-left: auto;
 		}
+		.rtl .wpcom-site-menu-intro-notice a.close-button {
+			margin-left: 0;
+			margin-right: auto;
+		}
```

Comment:
![badge](https://img.shields.io/badge/review-ask-blue.svg)

Would the following CSS be sufficient?

```suggestion
			margin-inline-start: auto;
		}
```

Reply:
Much better! 👍